protocol/0056-REWA-rewards_overview.md

```diff
@@ -486,10 +485,58 @@ Market goes above the threshold in trading value in an epoch before the reward a
     * Transfer 20000 USDC to `ETHUSDT | market creation | USDC` 
 
 #### Expectation
-At the end of the epoch 2 and at the end of epoch 3 no payout has been made for the market ETHUSDT and the reward account balances should remain unchanged.
+At the end of the epoch 3 a payout is made for the market ETHUSDT to the creator.
 
 
+### Distributing market creation rewards - multiple asset rewards (<a name="0056-REWA-043" href="#0056-REWA-043">0056-REWA-043</a>)
+#### Rationale 
+A market should be able to be rewarded multiple times if several reward pools are created with different payout assets.
 
+#### Setup
+* Setup a market ETHUSDT settling in USDT.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for `USDT` is `1`. 
+* Setup and fund multiple reward account for the market ETHUSDT:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During the epoch 2 let the traded value be greater than 10^6
+* During epoch 3, transfer 20000 USDC to `all | market creation | USDC` 
 
+#### Expectation
+At the end of epoch 2 the VEGA rewards should be distributed. Then, at the end of epoch 3, the funded USDC rewards should be distributed again to the market creator
+
+### Distributing market creation rewards - Same asset multiple party rewards (<a name="0056-REWA-044" href="#0056-REWA-044">0056-REWA-044</a>)
+#### Rationale 
+A market reward pool funded with the same asset by different parties should pay out to eligible markets as many times as there are parties, assuming threshold is reached.
+
+#### Setup
+* Setup a market ETHUSDT settling in USDT.
+* The value of `marketCreationQuantumMultiple` is `10^6` and `quantum` for `USDT` is `1`. 
+* Setup and fund multiple reward account for the market ETHUSDT:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+* start trading in the market such that traded value for fee purposes in USDT is less than 10^6
+* During the epoch 2 let the traded value be greater than 10^6
+* During epoch 3, setup and fund multiple reward account for the market ETHUSDT with a different party:
+    * Transfer 10000 $VEGA to `ETHUSDT | market creation | $VEGA` 
+
+#### Expectation
+At the end of epoch 2 the VEGA rewards should be distributed. Then, at the end of epoch 3, the VEGA rewards should again be distributed
```

Comment:
I think it's easier here to specify it in terms of balances:
end of epoch 2:
reward account balance = 0
proposer of ETHUSDT general balance in $VEGA = 10000k + extra from the infra fees of the transfers, not sure what the % is so can't tell an exact number here. 
end of epoch 3:
reward account balance = 0
proposer of ETHUSDT general balance in $VEGA = 20000k + 2 *extra from the infra fees of the transfers

end of epoch 4: (to show no more transfers take place)
reward account balance = 0
proposer of ETHUSDT general balance in $VEGA = 20000k + 2 *extra from the infra fees of the transfers